modules/ROOT/pages/setting-up-ldap-provider-for-spring-security.adoc

```diff
@@ -0,0 +1,118 @@
+= Configure LDAP Provider for Spring Security
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, connector, spring security, spring, ldap, authentication
+
+In Mule 4, you can use a Spring Security LDAP provider to perform component authorization, or use it as a Mule security provider via the Spring Module. Before you configure the LDAP provider obtain the Spring Jar files and declare Spring Beans.
+
+== Obtain JAR Files
+
+The Mule software distribution provides the Spring `.jar` files you need in the `<distribution>/lib/opt` directory:
+
+	* `spring-ldap-core-2.3.2.RELEASE.jar`
+
+	* `spring-security-config-4.2.6.RELEASE.jar`
+
+	* `spring-security-core-4.2.6.RELEASE.jar`
+
+	* `spring-security-ldap-4.2.6.RELEASE.jar`
+
+	* `spring-security-web-4.2.6.RELEASE.jar`
+
+== Declare Spring Beans
+
+The https://docs.spring.io/spring-security/site/docs/4.0.x/apidocs/org/springframework/security/ldap/DefaultSpringSecurityContextSource.html[DefaultSpringSecurityContextSource] is the access point for obtaining an LDAP context. +
```

Comment:
What is DefaultSpringSecurityContextSource? a parameter? something else?
"The DefaultSpringSecurityContextSource..." should be followed grammatically by a common noun object, such as parameter (or whatever it is). but I see that Spring uses this concatenation for its names? 
https://docs.spring.io/spring-security/site/docs/current/reference/htmlsingle/